bridges/DribbbleBridge.php

```diff
@@ -86,14 +86,23 @@ private function findJsonForShot($shot, $json){
 
 	private function getImageTag($preview_path, $title){
 		return sprintf(
-			'<br /> <a href="%s"><img src="%s" alt="%s" /></a>',
+			'<br /> <a href="%s"><img srcset="%s" alt="%s" /></a>',
 			$this->getFullSizeImagePath($preview_path),
 			$preview_path,
 			$title
 		);
 	}
 
 	private function getFullSizeImagePath($preview_path){
-		return explode('?compress=1', $preview_path)[0];
+		// Get last image from srcset
+		$src_set_urls = explode(',', $preview_path);
+		$url = $this->getArrayLastElement($src_set_urls);
+		$url = explode(' ', $url)[1];
+
+		return htmlspecialchars_decode($url);
+	}
+
+	private function getArrayLastElement($array) {
+		return array_values(array_slice($array, -1))[0];
```

Comment:
Maybe use [end](https://www.php.net/manual/ru/function.end.php) instead?

Reply:
Oh yeah thanks ! Fixed in 65b42bf